package.json

```diff
@@ -40,78 +38,70 @@
+    "color": "^3.1.3",
+    "fast-isnumeric": "^1.1.4",
+    "file-saver": "^2.0.5",
+    "highlight.js": "^11.0.1",
+    "moment": "^2.29.1",
+    "node-polyfill-webpack-plugin": "^1.1.4",
+    "plotly.js": "2.2.1",
+    "prop-types": "^15.7.2",
+    "ramda": "^0.27.1",
+    "rc-slider": "^9.7.2",
     "react-addons-shallow-compare": "^15.6.3",
-    "react-dates": "^20.1.0",
-    "react-docgen": "^3.0.0",
+    "react-dates": "^21.8.0",
+    "react-docgen": "^5.4.0",
     "react-dropzone": "^4.1.2",
-    "react-markdown": "^4.0.6",
-    "react-resize-detector": "^4.2.1",
+    "react-markdown": "^4.3.1",
+    "react-resize-detector": "^6.7.4",
     "react-select-fast-filter-options": "^0.2.3",
     "react-virtualized-select": "^3.1.3",
-    "uniqid": "^5.0.3"
+    "uniqid": "^5.3.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.13.0",
-    "@babel/core": "^7.4.0",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.10.4",
-    "@babel/plugin-proposal-object-rest-spread": "^7.4.0",
-    "@babel/plugin-proposal-optional-chaining": "^7.11.0",
-    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-    "@babel/preset-env": "^7.4.1",
-    "@babel/preset-react": "^7.0.0",
-    "@plotly/dash-component-plugins": "^1.1.0",
-    "@plotly/webpack-dash-dynamic-import": "^1.1.4",
-    "babel-eslint": "^10.0.1",
-    "babel-jest": "^24.5.0",
-    "babel-loader": "^8.0.5",
-    "check-prop-types": "^1.1.2",
-    "component-playground": "^3.0.0",
+    "@babel/cli": "^7.14.5",
+    "@babel/core": "^7.14.6",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/preset-env": "^7.14.7",
+    "@babel/preset-react": "^7.14.5",
+    "@plotly/dash-component-plugins": "^1.2.0",
+    "@plotly/webpack-dash-dynamic-import": "^1.2.0",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.2.2",
+    "component-playground": "^3.2.1",
     "copyfiles": "^2.4.1",
-    "css-loader": "^1.0.1",
-    "enzyme": "^3.7.0",
-    "enzyme-adapter-react-16": "^1.7.0",
-    "es-check": "^5.2.3",
-    "eslint": "^5.8.0",
-    "eslint-config-prettier": "^3.0.1",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-react": "^7.11.1",
-    "exec-sh": "^0.3.0",
-    "husky": "^4.2.3",
+    "css-loader": "^5.2.6",
+    "es-check": "^5.2.4",
+    "eslint": "^7.30.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-react": "^7.24.0",
+    "husky": "^7.0.1",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^24.5.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "^1.14.2",
-    "react": "16.13.0",
-    "react-dom": "16.13.0",
-    "react-jsx-parser": "^1.21.0",
-    "react-resize-detector": "^4.2.1",
-    "style-loader": "^0.23.1",
-    "styled-jsx": "^3.1.1",
-    "terser-webpack-plugin": "^2.3.0",
-    "webpack": "^4.37.0",
-    "webpack-cli": "^3.3.6",
-    "webpack-serve": "^2.0.3"
+    "prettier": "^2.3.2",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
+    "react-jsx-parser": "1.21.0",
```

Comment:
Pinned - upgrading `react-jsx-parser` to 1.28 breaks `dcc.Markdown` with html allowed.